### ISSUE TITLE
Cannot rate items

### ISSUE THREAD

**Issue by SimonHova** [ampache/python3-ampache#12]
I haven't had a chance to do some more serious troubleshooting on the Ampache side (like running any kind of debug logs), but when rating an item, rather than an HTML code, I just get a few blank lines, and the operation is unsuccessful. Can someone please validate this for me?

**Comment by lachlan-00** [comment]
i have an error in v4 of the api calls on develop which i'll fix but i'm not getting issues with rating on 5 calls.

empty responses are probably a PHP error on the server though.

**Comment by SimonHova** [comment]
Appreciated! I will try and turn on debugging and repost the logs.

**Comment by SimonHova** [comment]
Okay, I dug a bit deeper; the problem isn't with the rating but with the search function. Specifically, it returns blank results whenever I try to search for a song by MBID. So, if I send the function something like:

amp_song = ampache.advanced_search(args.Amp_URL, Amp_key, [['mbid',4,'06d3bb88-98ce-437f-9a04-d1bb21462f21']], object_type='song')

amp_song[0].text will return 1 (which means it found one result), but it will return a blank element in amp_song[1], breaking my rate function. What's odd is that the same code works perfectly with artist and albums.

**Comment by lachlan-00** [comment]
ah, good, i'll check tomorrow

**Comment by lachlan-00** [comment]
I'm def able to find the songs using xml and json. Is the call not returning anything in a browser?

it sounds like a null result.

here are 3 responses for mbid (song/album/artist) and they all seem okay. this is api5 are you using 4?

[artist.txt](https://github.com/ampache/python3-ampache/files/8218907/artist.txt)
[album.txt](https://github.com/ampache/python3-ampache/files/8218908/album.txt)
[song.txt](https://github.com/ampache/python3-ampache/files/8218909/song.txt)

**Comment by SimonHova** [comment]
It looks like it was a problem with me after all; I was trying to read the XML result as a text file and it was choking. I just modified my code and ran it through 500 songs and it rated them flawlessly. Though, I did think of an enhancement; I would prefer if it rated all the recordings with the MBID and not just the first one that it sees. I'll submit that one on the main ampache project later. Thanks again for your help!
